src/unix/into_file.rs

```diff
@@ -0,0 +1,15 @@
+#![cfg(feature = "std")]
+
+use crate::LockFile;
+use std::{fs::File, os::unix::io::FromRawFd as _};
+
+/// Turn the [`LockFile`] into a [`std::fs::File`]; you should probably also
+/// call [`crate::lockfile_truncate`].
+/// ```
+#[doc = include_str!("../../examples/lock_preserved.rs")]
+/// ```
+impl Into<File> for &mut LockFile {
+    fn into(self) -> File {
+        unsafe { File::from_raw_fd(self.raw()) }
+    }
+}
```

Comment:
As I said in other places: is this really required? Because this would also fall out of the scope of this feature.

Reply:
I think it's in-scope because if you can retain lockfiles you also are going to want to hold a single handle to them for reading/writing/seeking/all operations.